posthog/models/organization.py

```diff
@@ -148,8 +152,14 @@ def get_analytics_metadata(self):
         }
 
 
+@receiver(models.signals.pre_save, sender=Organization)
+def organization_about_to_be_created(sender, instance: Organization, raw, using, **kwargs):
+    if instance._state.adding:
+        instance.update_available_features()
```

Comment:
Maybe worth just updating this particular org?

Reply:
WDYM? That does seem to be the case already?